randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -197,3 +202,19 @@ def export_game(self, patch_data: dict, export_params: GameExportParams,
                 raise
             else:
                 raise RuntimeError(f"randomprime panic: {e}") from e
+
+        if random_enemy_attributes:
+            rand_scale_low = random_enemy_attributes.enemy_rando_range_scale_low
```

Comment:
Everything in newConfig is dicts, so this should be accessing via `[]`.
If no test failed because of it, then we need improvements to the tests

Reply:
It gives me "EnemyAttributeRandomizer" object is not subscriptable.